images.CI/build-image.ps1

```diff
@@ -54,6 +54,7 @@ packer build    -var "capture_name_prefix=$ResourcesNamePrefix" `
                 -var "virtual_network_name=$VirtualNetworkName" `
                 -var "virtual_network_resource_group_name=$VirtualNetworkRG" `
                 -var "virtual_network_subnet_name=$VirtualNetworkSubnet" `
+                -var "run_validation_diskspace=true" `
```

Comment:
What about customers who use our `Images.CI` scripts? Can we pass this variable from some environment variable?